packages/cli-template-monorepo-subgraph/apps/contracts/test/Feedback.ts

```diff
@@ -40,28 +47,38 @@ describe("Feedback", () => {
 
     describe("# sendFeedback", () => {
         it("Should allow users to send feedback anonymously", async () => {
-            const feedback = "Hello World"
+            const { semaphoreContract, feedbackContract, groupId } = await loadFixture(deployFeedbackFixture)
+
+            const users = [new Identity(), new Identity()]
+            const group = new Group()
+
+            for (const user of users) {
+                await feedbackContract.joinGroup(user.commitment)
+                group.addMember(user.commitment)
+            }
+
+            const feedback = encodeBytes32String("Hello World")
 
-            const proof = await generateProof(users[1], group, feedback, groupId)
+            const fullProof = await generateProof(users[1], group, feedback, groupId)
```

Comment:
After renaming `proof` to `points`, we started using `proof` rather than `fullProof`.